### PR TITLE
Datadog Update

### PR DESCRIPTION
**Pull request by MarkvanMents**
Buildpack work has been done, but changes for Developer Portal still need to be made.

Created branch to add missing new image before merging to development

### CODE REVIEW

**Comment by MarkvanMents** [comment]
Waiting for changes to environment variables in Developer Portal so that the settings can be passed to the buildpack.
Documentation of buildpack has been changed for customers building in their own environments.

**Comment by MarkvanMents** [comment]
Changes made in the back office, so documentation can be released.

File: content/developerportal/operate/datadog-metrics.md [hunk 1]
```diff
@@ -82,7 +80,9 @@ To send your runtime information to Datadog, you need to provide the Datadog API
 
 	This will ensure that some messages are sent to Datadog. You can change the log level later once you have confirmed that Datadog is receiving them.
 
-8.  Return to the **Environments** page for your app and *Deploy* or *Transport* your app into the selected environment.
+8. By default, the Datadog integration defaults to the US region (datadoghq.com). If you want to use a datadog site which is another region, set the `DD_SITE` environment variable to the required site. For example, for the EU Datadog site, set `DD_SITE` to `datadoghq.eu`.
```
Comment (markbastiaans) [inline content/developerportal/operate/datadog-metrics.md]: Datadog with a capital "D".

Comment (MarkvanMents) [reply content/developerportal/operate/datadog-metrics.md]: Well spotted. Changed.

**Pull request merged**